src/en/news/blog/2022/improving-dashboard-accessibility-for-blind-and-visually-impaired-ceph-users/index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Improving dashboard accessibility for blind and visually impaired ceph users"
+date: "2022-09-07"
+author: "Ngwa Sedrick Meh"
+image: "./images/banner.png"
+tags:
+  - "ceph"
+  - "dashboard"
+  - "accessibility"
+  - "a11y"
+  - "WCAG 2.1"
```

Comment:
The site requires all tags to be lowercase, like you've done with `accessibility`. I'd try `wcag2.1`, and if it still has issues, remove the tag altogether.